locales/en-US/strings.properties

```diff
@@ -205,3 +205,6 @@ firstrun_privacy_notice=Privacy Notice
 
 firstrun_continue_to_login=Continue
 firstrun_skip_login=Skip this step
+
+# Tooltip for context menu
+context_menu_title=Menu
```

Comment:
I believe typically these are "actions," e.g,. "Open Context Menu" I'll check with UX in the bug.

Also, I'm not sure if we want the same title for both cards and the section menu, but I suppose if so or different, the section menu title could be fixed in a separate PR.

Reply:
I asked Aaron and he said "Menu" was good for all